commit b8a6b494ede7fc4b6efeea15daae3124c2aed54b
Author: Laurent Bigonville <bigon@bigon.be>
Date:   Mon Sep 24 14:17:32 2012 +0200

    Split libauparse out of libaudit package and put /etc/libaudit.conf in its own package thanks to Alban Browaeys (Closes: #682251)

diff --git a/debian/changelog b/debian/changelog
index 9435aa6..10980d1 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+audit (1:2.2.1-2) UNRELEASED; urgency=low
+
+  * Split libauparse out of libaudit package and put /etc/libaudit.conf in its
+    own package thanks to Alban Browaeys (Closes: #682251)
+
+ -- Laurent Bigonville <bigon@debian.org>  Mon, 24 Sep 2012 14:14:00 +0200
+
 audit (1:2.2.1-1) experimental; urgency=low
 
   * Non-maintainer upload (with maintainer's blessing)
diff --git a/debian/control b/debian/control
index d6192b7..bf2f3fe 100644
--- a/debian/control
+++ b/debian/control
@@ -26,15 +26,53 @@ Description: User space tools for security auditing
  .
  Also contains the audit dispatcher "audisp".
 
-Package: libaudit1
+Package: libauparse0
 Architecture: any
 Priority: optional
 Depends: ${shlibs:Depends}, ${misc:Depends}
+Breaks: libaudit0, libaudit1 (<< 1:2.2.1-2)
+Replaces: libaudit0, libaudit1 (<< 1:2.2.1-2)
+Description: Dynamic library for parsing security auditing
+ The libauparse package contains the dynamic libraries needed for
+ applications to use the audit framework. It is used to monitor systems for
+ security related events.
+ .
+ This package contains the libauparse0 library.
+
+Package: libauparse-dev
+Section: libdevel
+Architecture: any
+Depends: libauparse0 (= ${binary:Version}), ${misc:Depends}
+Breaks: libaudit-dev (<< 1:2.2.1-2)
+Replaces: libaudit-dev (<< 1:2.2.1-2)
+Description: Header files and static library for the libauparse0 library
+ The audit-libs parse package contains the dynamic libraries needed for
+ applications to use the audit framework. It is used to monitor systems for
+ security related events.
+
+Package: libaudit1
+Architecture: any
+Priority: optional
+Depends: ${shlibs:Depends}, ${misc:Depends}, libaudit-common (= ${source:Version})
 Description: Dynamic library for security auditing
  The audit-libs package contains the dynamic libraries needed for
  applications to use the audit framework. It is used to monitor systems for
  security related events.
 
+Package: libaudit-common
+Architecture: all
+Priority: optional
+Depends: ${misc:Depends}
+Breaks: libaudit0, libaudit1 (<< 1:2.2.1-2)
+Replaces: libaudit0, libaudit1 (<< 1:2.2.1-2)
+Description: Dynamic library for security auditing - common files
+ The audit-libs package contains the dynamic libraries needed for
+ applications to use the audit framework. It is used to monitor systems for
+ security related events.
+ .
+ This package contains the libaudit.conf configuration file and the associated
+ manpage.
+
 Package: libaudit-dev
 Section: libdevel
 Architecture: any
diff --git a/debian/libaudit-common.install b/debian/libaudit-common.install
new file mode 100644
index 0000000..bec11e7
--- /dev/null
+++ b/debian/libaudit-common.install
@@ -0,0 +1,2 @@
+etc/libaudit.conf
+usr/share/man/man5/libaudit.conf.5
diff --git a/debian/libaudit-dev.install b/debian/libaudit-dev.install
index 71ce6a7..023fc32 100644
--- a/debian/libaudit-dev.install
+++ b/debian/libaudit-dev.install
@@ -1,4 +1,5 @@
 usr/lib/libaudit.a
-usr/lib/libauparse.a
-usr/include
-usr/share/man/man3
+usr/include/libaudit.h
+usr/share/man/man3/audit*
+usr/share/man/man3/set_aumessage_mode.3
+usr/share/man/man3/get_auditfail_action.3
diff --git a/debian/libaudit-dev.links b/debian/libaudit-dev.links
index 2e2a4b5..6ce6b20 100644
--- a/debian/libaudit-dev.links
+++ b/debian/libaudit-dev.links
@@ -1,2 +1 @@
 lib/libaudit.so.0	usr/lib/libaudit.so
-lib/libauparse.so.0	usr/lib/libauparse.so
diff --git a/debian/libaudit1.install b/debian/libaudit1.install
index 2075355..7b4e6ed 100644
--- a/debian/libaudit1.install
+++ b/debian/libaudit1.install
@@ -1,3 +1 @@
-usr/lib/libaudit.so.*	lib
-usr/lib/libauparse.so.*	lib
-etc/libaudit.conf
+usr/lib/libaudit.so.* lib
diff --git a/debian/libaudit1.symbols b/debian/libaudit1.symbols
index b3fb6f5..1d1af3e 100644
--- a/debian/libaudit1.symbols
+++ b/debian/libaudit1.symbols
@@ -3,115 +3,65 @@ libaudit.so.1 libaudit1 #MINVER#
  _audit_elf@Base 1:2.2.1
  _audit_permadded@Base 1:2.2.1
  _audit_syscalladded@Base 1:2.2.1
- audit_action_to_name@Base 1.7.13
- audit_add_dir@Base 1.7.13
-#MISSING: 1:2.2.1-1# audit_add_rule@Base 1.7.13
- audit_add_rule_data@Base 1.7.13
- audit_add_watch@Base 1.7.13
- audit_add_watch_dir@Base 1.7.13
- audit_close@Base 1.7.13
-#MISSING: 1:2.2.1-1# audit_delete_rule@Base 1.7.13
- audit_delete_rule_data@Base 1.7.13
- audit_detect_machine@Base 1.7.13
- audit_elf_to_machine@Base 1.7.13
- audit_encode_nv_string@Base 1.7.13
- audit_encode_value@Base 1.7.13
- audit_errno_to_name@Base 1.7.13
- audit_field_to_name@Base 1.7.13
- audit_flag_to_name@Base 1.7.13
- audit_ftype_to_name@Base 1.7.13
- audit_get_reply@Base 1.7.13
- audit_getloginuid@Base 1.7.13
- audit_is_enabled@Base 1.7.13
- audit_log_acct_message@Base 1.7.13
-#MISSING: 1:2.2.1-1# audit_log_avc@Base 1.7.13
- audit_log_semanage_message@Base 1.7.13
- audit_log_user_avc_message@Base 1.7.13
- audit_log_user_comm_message@Base 1.7.13
- audit_log_user_command@Base 1.7.13
- audit_log_user_message@Base 1.7.13
- audit_machine_to_elf@Base 1.7.13
- audit_machine_to_name@Base 1.7.13
- audit_make_equivalent@Base 1.7.13
+ audit_action_to_name@Base 1:2.2.1
+ audit_add_dir@Base 1:2.2.1
+ audit_add_rule_data@Base 1:2.2.1
+ audit_add_watch@Base 1:2.2.1
+ audit_add_watch_dir@Base 1:2.2.1
+ audit_close@Base 1:2.2.1
+ audit_delete_rule_data@Base 1:2.2.1
+ audit_detect_machine@Base 1:2.2.1
+ audit_elf_to_machine@Base 1:2.2.1
+ audit_encode_nv_string@Base 1:2.2.1
+ audit_encode_value@Base 1:2.2.1
+ audit_errno_to_name@Base 1:2.2.1
+ audit_field_to_name@Base 1:2.2.1
+ audit_flag_to_name@Base 1:2.2.1
+ audit_ftype_to_name@Base 1:2.2.1
+ audit_get_reply@Base 1:2.2.1
+ audit_getloginuid@Base 1:2.2.1
+ audit_is_enabled@Base 1:2.2.1
+ audit_log_acct_message@Base 1:2.2.1
+ audit_log_semanage_message@Base 1:2.2.1
+ audit_log_user_avc_message@Base 1:2.2.1
+ audit_log_user_comm_message@Base 1:2.2.1
+ audit_log_user_command@Base 1:2.2.1
+ audit_log_user_message@Base 1:2.2.1
+ audit_machine_to_elf@Base 1:2.2.1
+ audit_machine_to_name@Base 1:2.2.1
+ audit_make_equivalent@Base 1:2.2.1
  audit_msg@Base 1:2.2.1
- audit_msg_type_to_name@Base 1.7.13
- audit_name_to_action@Base 1.7.13
- audit_name_to_errno@Base 1.7.13
- audit_name_to_field@Base 1.7.13
- audit_name_to_flag@Base 1.7.13
- audit_name_to_ftype@Base 1.7.13
- audit_name_to_machine@Base 1.7.13
- audit_name_to_msg_type@Base 1.7.13
- audit_name_to_syscall@Base 1.7.13
- audit_number_to_errmsg@Base 1.7.13
- audit_open@Base 1.7.13
- audit_operator_to_symbol@Base 1.7.13
-#MISSING: 1:2.2.1-1# audit_request_rules_list@Base 1.7.13
- audit_request_rules_list_data@Base 1.7.13
- audit_request_signal_info@Base 1.7.13
- audit_request_status@Base 1.7.13
-#MISSING: 1:2.2.1-1# audit_rule_fieldpair@Base 1.7.13
- audit_rule_fieldpair_data@Base 1.7.13
-#MISSING: 1:2.2.1-1# audit_rule_free@Base 1.7.13
- audit_rule_free_data@Base 1.7.13
+ audit_msg_type_to_name@Base 1:2.2.1
+ audit_name_to_action@Base 1:2.2.1
+ audit_name_to_errno@Base 1:2.2.1
+ audit_name_to_field@Base 1:2.2.1
+ audit_name_to_flag@Base 1:2.2.1
+ audit_name_to_ftype@Base 1:2.2.1
+ audit_name_to_machine@Base 1:2.2.1
+ audit_name_to_msg_type@Base 1:2.2.1
+ audit_name_to_syscall@Base 1:2.2.1
+ audit_number_to_errmsg@Base 1:2.2.1
+ audit_open@Base 1:2.2.1
+ audit_operator_to_symbol@Base 1:2.2.1
+ audit_request_rules_list_data@Base 1:2.2.1
+ audit_request_signal_info@Base 1:2.2.1
+ audit_request_status@Base 1:2.2.1
+ audit_rule_fieldpair_data@Base 1:2.2.1
+ audit_rule_free_data@Base 1:2.2.1
  audit_rule_interfield_comp_data@Base 1:2.2.1
-#MISSING: 1:2.2.1-1# audit_rule_syscall@Base 1.7.13
- audit_rule_syscall_data@Base 1.7.13
-#MISSING: 1:2.2.1-1# audit_rule_syscallbyname@Base 1.7.13
- audit_rule_syscallbyname_data@Base 1.7.13
+ audit_rule_syscall_data@Base 1:2.2.1
+ audit_rule_syscallbyname_data@Base 1:2.2.1
  audit_send@Base 1:2.2.1
- audit_send_user_message@Base 1.7.13
- audit_set_backlog_limit@Base 1.7.13
- audit_set_enabled@Base 1.7.13
- audit_set_failure@Base 1.7.13
- audit_set_pid@Base 1.7.13
- audit_set_rate_limit@Base 1.7.13
- audit_setloginuid@Base 1.7.13
- audit_syscall_to_name@Base 1.7.13
- audit_trim_subtrees@Base 1.7.13
- audit_update_watch_perms@Base 1.7.13
- audit_value_needs_encoding@Base 1.7.13
- get_auditfail_action@Base 1.7.13
- set_aumessage_mode@Base 1.7.13
-libauparse.so.0 libaudit1 #MINVER#
- auparse_add_callback@Base 1.7.13
- auparse_destroy@Base 1.7.13
- auparse_feed@Base 1.7.13
- auparse_find_field@Base 1.7.13
- auparse_find_field_next@Base 1.7.13
- auparse_first_field@Base 1.7.13
- auparse_first_record@Base 1.7.13
- auparse_flush_feed@Base 1.7.13
- auparse_get_field_int@Base 1.7.13
- auparse_get_field_name@Base 1.7.13
- auparse_get_field_str@Base 1.7.13
- auparse_get_field_type@Base 1.7.13
- auparse_get_filename@Base 1.7.13
- auparse_get_line_number@Base 1.7.13
- auparse_get_milli@Base 1.7.13
- auparse_get_node@Base 1.7.13
- auparse_get_num_fields@Base 1.7.13
- auparse_get_num_records@Base 1.7.13
- auparse_get_record_text@Base 1.7.13
- auparse_get_serial@Base 1.7.13
- auparse_get_time@Base 1.7.13
- auparse_get_timestamp@Base 1.7.13
- auparse_get_type@Base 1.7.13
- auparse_goto_record_num@Base 1.7.13
- auparse_init@Base 1.7.13
- auparse_interpret_field@Base 1.7.13
- auparse_next_event@Base 1.7.13
- auparse_next_field@Base 1.7.13
- auparse_next_record@Base 1.7.13
- auparse_node_compare@Base 1.7.13
- auparse_reset@Base 1.7.13
- auparse_timestamp_compare@Base 1.7.13
- auparse_type_t@Base 1.7.13
- ausearch_add_expression@Base 1.7.13
- ausearch_add_interpreted_item@Base 1.7.13
- ausearch_add_item@Base 1.7.13
- ausearch_add_regex@Base 1.7.13
- ausearch_add_timestamp_item@Base 1.7.13
- ausearch_clear@Base 1.7.13
- ausearch_next_event@Base 1.7.13
- ausearch_set_stop@Base 1.7.13
+ audit_send_user_message@Base 1:2.2.1
+ audit_set_backlog_limit@Base 1:2.2.1
+ audit_set_enabled@Base 1:2.2.1
+ audit_set_failure@Base 1:2.2.1
+ audit_set_pid@Base 1:2.2.1
+ audit_set_rate_limit@Base 1:2.2.1
+ audit_setloginuid@Base 1:2.2.1
+ audit_syscall_to_name@Base 1:2.2.1
+ audit_trim_subtrees@Base 1:2.2.1
+ audit_update_watch_perms@Base 1:2.2.1
+ audit_value_needs_encoding@Base 1:2.2.1
+ get_auditfail_action@Base 1:2.2.1
+ set_aumessage_mode@Base 1:2.2.1
diff --git a/debian/libauparse-dev.install b/debian/libauparse-dev.install
new file mode 100644
index 0000000..8bd4794
--- /dev/null
+++ b/debian/libauparse-dev.install
@@ -0,0 +1,5 @@
+usr/lib/libauparse.a
+usr/include/auparse.h
+usr/include/auparse-defs.h
+usr/share/man/man3/auparse*
+usr/share/man/man3/ausearch*
diff --git a/debian/libauparse-dev.links b/debian/libauparse-dev.links
new file mode 100644
index 0000000..2625b4a
--- /dev/null
+++ b/debian/libauparse-dev.links
@@ -0,0 +1 @@
+lib/libauparse.so.0	usr/lib/libauparse.so
diff --git a/debian/libauparse0.install b/debian/libauparse0.install
new file mode 100644
index 0000000..d1d88ff
--- /dev/null
+++ b/debian/libauparse0.install
@@ -0,0 +1 @@
+usr/lib/libauparse.so.* lib
diff --git a/debian/libauparse0.symbols b/debian/libauparse0.symbols
new file mode 100644
index 0000000..fdba57b
--- /dev/null
+++ b/debian/libauparse0.symbols
@@ -0,0 +1,42 @@
+libauparse.so.0 libauparse0 #MINVER#
+ auparse_add_callback@Base 1:2.2.1
+ auparse_destroy@Base 1:2.2.1
+ auparse_feed@Base 1:2.2.1
+ auparse_find_field@Base 1:2.2.1
+ auparse_find_field_next@Base 1:2.2.1
+ auparse_first_field@Base 1:2.2.1
+ auparse_first_record@Base 1:2.2.1
+ auparse_flush_feed@Base 1:2.2.1
+ auparse_get_field_int@Base 1:2.2.1
+ auparse_get_field_name@Base 1:2.2.1
+ auparse_get_field_str@Base 1:2.2.1
+ auparse_get_field_type@Base 1:2.2.1
+ auparse_get_filename@Base 1:2.2.1
+ auparse_get_line_number@Base 1:2.2.1
+ auparse_get_milli@Base 1:2.2.1
+ auparse_get_node@Base 1:2.2.1
+ auparse_get_num_fields@Base 1:2.2.1
+ auparse_get_num_records@Base 1:2.2.1
+ auparse_get_record_text@Base 1:2.2.1
+ auparse_get_serial@Base 1:2.2.1
+ auparse_get_time@Base 1:2.2.1
+ auparse_get_timestamp@Base 1:2.2.1
+ auparse_get_type@Base 1:2.2.1
+ auparse_goto_record_num@Base 1:2.2.1
+ auparse_init@Base 1:2.2.1
+ auparse_interpret_field@Base 1:2.2.1
+ auparse_next_event@Base 1:2.2.1
+ auparse_next_field@Base 1:2.2.1
+ auparse_next_record@Base 1:2.2.1
+ auparse_node_compare@Base 1:2.2.1
+ auparse_reset@Base 1:2.2.1
+ auparse_timestamp_compare@Base 1:2.2.1
+ auparse_type_t@Base 1:2.2.1
+ ausearch_add_expression@Base 1:2.2.1
+ ausearch_add_interpreted_item@Base 1:2.2.1
+ ausearch_add_item@Base 1:2.2.1
+ ausearch_add_regex@Base 1:2.2.1
+ ausearch_add_timestamp_item@Base 1:2.2.1
+ ausearch_clear@Base 1:2.2.1
+ ausearch_next_event@Base 1:2.2.1
+ ausearch_set_stop@Base 1:2.2.1
diff --git a/debian/rules b/debian/rules
index 3bc6c2d..5905bf2 100755
--- a/debian/rules
+++ b/debian/rules
@@ -98,7 +98,7 @@ override_dh_makeshlibs:
 	chrpath -d debian/auditd/usr/bin/aulast
 	chrpath -d debian/auditd/usr/bin/aulastlog
 	chrpath -d debian/auditd/usr/bin/ausyscall
-	chrpath -d debian/libaudit1/lib/libauparse.so.0.0.0
+	chrpath -d debian/libauparse0/lib/libauparse.so.0.0.0
 	chrpath -d debian/python-audit/usr/lib/python*/*-packages/_audit.so
 	chrpath -d debian/python-audit/usr/lib/python*/*-packages/auparse.so
 	dh_makeshlibs
